requirements.txt

```diff
@@ -6,7 +6,7 @@ marshmallow-jsonapi==0.15.1
 marshmallow==2.15.1
 setuptools>=40.0.0
 argcomplete==1.12.1
-indexed_gzip==1.5.3
+indexed_gzip_fileobj_fork_epicfaace==1.5.4
```

Comment:
This fork includes a fix for https://github.com/codalab/codalab-worksheets/issues/3514.